inventory/group_vars/all.yml

```diff
@@ -71,8 +72,19 @@ eessi_cvmfs_repositories:
       - CVMFS_AUTO_GC=false
       - CVMFS_AUTO_TAG_TIMESPAN="30 days ago"
       - CVMFS_GARBAGE_COLLECTION=true
-    client_options:
-      - CVMFS_NFILES=4096
+    client_options: []
+#      - CVMFS_NFILES=4096
```

Comment:
Why was this commented out?

Reply:
Good question! :-)
To be honest, I wasn't even sure why it was there in the first place (it's a client-specific setting), and I don't see it in any other config packages, so I've commented these lines. I'll just remove them.

Reply:
Probably just copied from https://github.com/galaxyproject/ansible-cvmfs/blob/master/README.md#example-playbook...